app/assets/javascripts/the_bridge_ng/app.js.coffee

```diff
@@ -77,6 +77,9 @@ Array::filter = (func) -> x for x in @ when func(x)
     .when('/welcome', {
       templateUrl: '../the_bridge_templates/static/welcome2.html'
     })
+    .when('/vote/:id', {
+      templateUrl: '../the_bridge_templates/vote.html',
```

Comment:
Why? There is no need to show votes on their own, and there is no vote.html file in the templates anyway.

Please remove.